docs/UserGuide.md

```diff
@@ -544,21 +525,50 @@ Listed here are some frequently asked questions. Should you require any other as
+| **`PHONE`**             | `p/`                 | Should contain only **numbers**, and it should be at least 3 digits and at most 15 digits long.                                                                                                                                                                                                                                                                                                  |
+| **`EMAIL`**             | `e/`                 | Should contain only **alphanumeric characters and 1 `@` character**.<br/>The local segment (before `@`) should be at most 64 characters long, and the domain segment (after `@`) should be at most 255 characters long.<br/>Special characters (`+`, `_`, `.`, `-`) can be included but **not as the first or last characters** of any segment. They also cannot be used adjacent to each other. |
+| **`ROOM`**              | `r/`                 | Should contain only **alphanumeric characters and hyphens (`-`)**.<br/>A hyphen can only be added if it is in between 2 alphanumeric characters.                                                                                                                                                                                                                                                 |
+| **`DATE_RANGE`**        | `dr/`                | Should follow the format `dd/MM/yy - dd/MM/yy` with 2 valid dates and the second date should be later than the first date.                                                                                                                                                                                                                                                                       |
+| **`NUMBER_OF_GUESTS`**  | `ng/`                | Should contain only **numbers**, and be at least 1 and at most 4 numbers long.                                                                                                                                                                                                                                                                                                                   |
+| **`REQUEST`**           | `rq/`                | Is an **optional field**, and should be at most 500 characters long.<br/>Should not contain any field prefixes after whitespace in its content (e.g., `rq/Extra bed & p/c`).                                                                                                                                                                                                                     |
```

Comment:
Same comment above for request subsection under Features.

Reply:
Added!